digitalpy/core/digipy_configuration/impl/config_action_key_provider.py

```diff
@@ -1,5 +1,5 @@
-from digitalpy.config.action_key_provider import ActionKeyProvider
-from digitalpy.config.configuration import Configuration
+from digitalpy.core.digipy_configuration.action_key_provider import ActionKeyProvider
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position